lib/core/src/Data/Quantity.hs

```diff
@@ -117,6 +121,9 @@ instance FromText b => FromText (Quantity sym b) where
 instance ToText b => ToText (Quantity sym b) where
     toText (Quantity b) = toText b
 
+instance Buildable b => Buildable (Quantity sym b) where
+    build (Quantity b) = build b
```

Comment:
That defies a bit the purpose... At the very least, we should show the unit!

Reply:
Moving to separate PR.